scripts/core/exp-usm-import-release.yml

```
@@ -0,0 +1,42 @@
+--- #--------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi USM Import/Release Extension APIs"
+ordinal: "0"
```

```suggestion
ordinal: "1"
```
This ensures that this code will be placed after the `ur_handle_*` definitions

---

Done.